### PR TITLE
Modify screenshots on change of background ID

### PR DESCRIPTION
**Pull request by jaydhulia**

### CODE REVIEW

File: app/public/templates/scene.html [hunk 1]
```diff
@@ -190,22 +195,26 @@
           {% end %}
           inspector.openInspector();
       });
-      async function takeSceneScreenshot(){
-          const screenshot_canvas = scene.components.screenshot.getCanvas('perspective');
-          await uploadScreenshotToS3(screenshot_canvas.toDataURL("image/jpeg"));
+      async function takeSceneScreenshot(s3_key=""){
+        if(initial_load_js || (!initial_load_js && s3_key !== "")){
```
Comment (drbalar) [inline app/public/templates/scene.html:199]: Minor thing, can we make this into `if(initial_load_js || s3_key !== ""){`?
Unless you kept it like this intentionally for readability

**Review by drbalar — APPROVED** [review]
looks good, just a comment

File: app/public/templates/scene.html (hunk 1 above)
Comment (ahmedhamodi) [reply app/public/templates/scene.html:199]: Agreed. From an outside perspective, I think `initial_load_js || (!initial_load_js && s3_key !== "")` is less readable than what Dhruvin suggested here. And yeah, appears logically equivalent to me.

File: app/public/templates/scene.html [hunk 2]
```diff
@@ -237,6 +248,10 @@
           };
           xhr.send(JSON.stringify(requestData));
       }
+      function getCookieScene(name) {
+          let r = document.cookie.match("\\b" + name + "=([^;]*)\\b");
+          return r ? r[1] : undefined;
+        }
```
Comment (ahmedhamodi) [inline app/public/templates/scene.html:254]: The nesting of function and the closing } aren't the same ( the } appears to be 2 spaces deeper ).

**Review by drbalar — APPROVED** [review]
Looks good 🎉 🚀

**Pull request merged**